content/blog/2023/11/2023-11-29-jenkins-to-github.adoc

```diff
@@ -0,0 +1,269 @@
+---
+layout: post
+title: "Log from Jenkins Jobs to GitHub Pull Requests as Checks"
+tags:
+- jenkins
+- github
+author: tslmy
+description: "An overview about methods by which you can log stuff from a Jenkins Job to a GitHub PR as “Checks”."
+note: "A guest post, originally posted at https://lmy.medium.com/log-from-jenkins-jobs-to-github-pull-requests-as-checks-fd8b57b91a1d[lmy.medium.com]."
+---
+
+You have a *GitHub repo* and a *Jenkins server*.
+The Jenkins has Jobs for running unit tests, computing code coverages, and executing static analyses.
```

Comment:
"Job" is a common noun, not a proper noun, so there is no reason to capitalize it.